src/main/java/com/nirima/jenkins/plugins/docker/action/DockerBuildImageAction.java

```diff
@@ -72,6 +92,15 @@ public DescriptorImpl getDescriptor() {
         return (DescriptorImpl) Jenkins.getInstance().getDescriptorOrDie(getClass());
     }
 
+    public void setPull(boolean pull) {
```

Comment:
Minor point: I'd prefer field getters/setters to immediately follow the constructor(s).
(I'm not going to refuse to merge the code if you don't do this, but if it's no hassle for you to move it, please do so)